result_test.go

```diff
@@ -170,34 +138,6 @@ func TestNewResultObjectErrors(t *testing.T) {
 			err: `cannot return errors from dig.Out, return it from the constructor instead: field "Error" (error)`,
 		},
 		{
-			desc: "type conflict",
-			give: struct {
-				Out
-
-				Reader io.Reader
-				Writer io.Writer
-
-				Nested struct {
-					Out
-
-					AnotherReader io.Reader
-					AnotherWriter io.Writer `name:"conflict-free-writer"`
-				}
-			}{},
-			err: "returns multiple io.Reader",
-		},
-		{
-			desc: "name conflict",
-			give: struct {
-				Out
-
-				Reader        io.Reader
-				NamedWriter   io.Writer `name:"what"`
-				AnotherWriter io.Writer `name:"what"`
-			}{},
-			err: "returns multiple io.Writer:what",
```

Comment:
why is this case deleted?

Reply:
This validation is no longer done in newResult. Instead it's done in connectionVisitor.
Plus these error paths are already exercised by the integration tests.